feature-libs/order/components/myaccount-order-view/myaccount-order-view.component.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, inject, OnDestroy } from '@angular/core';
+import { Product } from '@spartacus/core';
+import { OrderHistoryExtendedViewService } from '@spartacus/order/core';
+import { Order, OrderHistoryListView } from '@spartacus/order/root';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-myaccount-order-view',
+  templateUrl: './myaccount-order-view.component.html',
+})
+export class MyAccountOrderViewComponent implements OnDestroy {
```

Comment:
[MEDIUM]
We display many orders in this component. What about using plural form?
`...OrderComponent` vs `...OrdersComponent` (+ adjust component selector, filename)

Reply:
done